region/crateria/west/Gauntlet Entrance.json

```diff
@@ -214,6 +214,7 @@
       "exitCondition": {
         "leaveWithGModeSetup": {}
       },
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}],
```

Comment:
Good example. In this case what we can do is change the `unlocksDoors` in that PB strat to have `useImplicitRequires: "false"` for the powerbomb `unlocksDoors`, so this way it represents that an extra PB isn't needed to unlock the door since as part of the strat you will already be using a Power Bomb that reaches the door.

Even in Map Rando this could have an application in the very niche situation where you bypassed the lock on that door on the way in, allowing you to save a Power Bomb.